app/scripts/controllers/HomeCtrl.js

```diff
@@ -1,16 +1,15 @@
 (function() {
-  function HomeCtrl(ApiRequests, $cookies) {
-      this.ApiRequests = ApiRequests;
+  function HomeCtrl(ListsApiRequests, $cookies, UsersApiRequests) {
+      UsersApiRequests.user_signed_in();
 
-      ApiRequests.user_signed_in();
+      this.ListsApiRequests = ListsApiRequests;
 
       this.pass_list_id = function(list_id) {
           $cookies.put('blocitoffListId', list_id);
-          ApiRequests.task_return();
       }
 
       this.delete_list = function(list_id) {
-        ApiRequests.delete_list(list_id);
+        ListsApiRequests.delete_list(list_id);
```

Comment:
JavaScript follows [Camel Case](https://en.wikipedia.org/wiki/Camel_case) for naming variables and functions. If you switch your functions to follow: `deleteList` and such, it'll look a lot more consistent.